prometheus/prometheus.go

```diff
@@ -11,6 +11,7 @@ import (
 
 func HandleHttp(addr string, chErr chan error) {
 	http.Handle("/metrics", promhttp.Handler())
-	log.WithFields(log.Fields{"address" : fmt.Sprintf("http://%s/metrics", addr)}).Info("Serving metrics")
+	log.WithFields(log.Fields{
+		"address" : fmt.Sprintf("http://%s/metrics", addr)}).Info("Serving metrics")
```

Comment:
What's missing here?

Reply:
Or – even better – what's too much here?

Reply:
corrected it [here](https://github.com/Icinga/icingadb/pull/76/commits/49f2fd0ee621918d744d67f179f142873fe41f75)

Reply:
OK, fine.